src/wp-includes/blocks.php

```diff
@@ -1219,6 +1219,8 @@ function update_ignored_hooked_blocks_postmeta( $post ) {
 
 	if ( 'wp_navigation' === $post->post_type ) {
 		$wrapper_block_type = 'core/navigation';
+	} elseif ( 'wp_block' === $post->post_type ) {
```

Comment:
Should mention `wp_block` in `@since` annotation like `wp_navigation`. See [[59523]](https://core.trac.wordpress.org/changeset/59523)

Reply:
Hmm, this should already be covered by the existing `@since` : https://github.com/WordPress/wordpress-develop/blob/816ec5df8a5970254d4b6d6ab73ba01b06d3c6f4/src/wp-includes/blocks.php#L1180

(Note that it says _non_-`wp_navigation` post types: Before 6.8, it was just `wp_navigation`; now, other post types are allowed as well. We did have to finetune the logic for `wp_block` in this PR, but that probably doesn't need to be documented separately in the `@since` line :)